core/templates/dev/head/pages/skill_editor/editor_tab/SkillConceptCardEditorDirective.js

```diff
@@ -0,0 +1,177 @@
+// Copyright 2018 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Directive for the concept card editor.
+ */
+
+oppia.directive('skillConceptCardEditor', [
+  'UrlInterpolationService', 'SkillUpdateService', 'SkillEditorStateService',
+  function (
+      UrlInterpolationService, SkillUpdateService, SkillEditorStateService) {
+    return {
+      restrict: 'E',
+      scope: {},
+      templateUrl: UrlInterpolationService.getDirectiveTemplateUrl(
+        '/pages/skill_editor/editor_tab/' +
+        'skill_concept_card_editor_directive.html'),
+      controller: [
+        '$scope', '$filter', '$uibModal',
+        function($scope, $filter, $uibModal) {
+          $scope.skill = SkillEditorStateService.getSkill();
+          $scope.dragDotsImgUrl = UrlInterpolationService.getStaticImageUrl(
```

Comment:
I think this can be deleted right?

Reply:
I think it's being used by the worked examples? I'm deleting the one in misconceptions editor though

Reply:
Ah I see ok. I probably misidentified the one that needed to be deleted, thanks!